### DISCUSSION TITLE
Intents - Implement 'stop cover' intent and sentences (during opening or closing cover)

### DISCUSSION THREAD

**Discussion by andrejs2** [OHF-Voice/intents#2056] [_Ideas]
Proposal for implementing `coverStop` intent and corresponding sentences.

When the sentence is called: '`open the blinds`'  or '`close the blinds`' the HA starts opening the `blinds`.

Adding sentences and intent for `cover `to `stop `when `opening `or `closing `(e.g. `coverHassStop`) will `stop opening` / `closing` cover.

This intent can be useful with `cover `which takes some time to `open`/`close `and will allow to` stop cover` at desired position (e.g. sun, shade position,...) in no relation to `HassSetPosition` intent (when desired `position` in % could be a bare guessing for a user).

Thanks!

**Comment by maxdd** [comment]
When is this planned? I went through the hassle of setting up cover entities to then figure out i cant stop them halfway

**Comment by dekiesel** [comment]
This would be great to have!
